props/saber_sabersense_buttons.h

```diff
@@ -720,7 +741,7 @@ class SabersenseButtons : public PROP_INHERIT_PREFIX PropBase {
   }
 #endif
 
-  void NextBladeArray() {
+  void IdentBladeArray() {
```

Comment:
This seems like a step down in clarity.
What does "identBladeArray" mean? What does this function do now?